assets/javascript/app.js

```diff
@@ -0,0 +1,154 @@
+$(document).ready(function () {
+    var options = [
+        {
+            question: "What is the capture device used to contain Pokemon?", 
+            choice: ["Pokebox", "Pokeball", "Pokesphere", "Pokenet"],
+            answer: 1
+         },
+         {
+            question: "Who is Spongebob's bestfriend?", 
+            choice: ["Patrick", "Sandy", "Magic Mike", "Rick Flair"],
+            answer: 0
+         }, 
+         {
+            question: "What is the name of the protagonist in 'How I Met Your Mother'?", 
+            choice: ["Ned Mosby", "Ted Crosby", "Ted Mosby", "Vanilla Ice" ],
+            answer: 2
+        }, 
+        {
+            question: "Which fictional creature has the title of 'King Of All Monsters'?", 
+            choice: ["Rodan", "Chidorah", "Godzilla", "Destroyah"],
+            answer: 2
+        }, 
+        {
+            question: "How many Infinity Stones are there in the Marvel Universe?", 
+            choice: ["12", "5", "9", "6"],
+            answer: 3
+        }, 
+        {
+            question: "In the show,'Family Guy', what is the name of the Griffin's dog?", 
+            choice: ["Scooby", "Brian", "Bryan", "Marmaduke" ],
+            answer: 1
+        }, 
+        {
+            question: "What is Goku's race?", 
+            choice: ["Namekian", "Saiyan", "Human", "Ape"],
+            answer: 1
+        }, 
+        {
+            question: "What element can Magneto control?", 
+            choice: ["Metal", "Fire", "Water", "Air"],
+            answer: 0
+        }];
+    
+    var correctCount = 0;
+    var wrongCount = 0;
+    var unanswerCount = 0;
+    var timer = 20;
+    var intervalId;
+    var userGuess ="";
+    var running = false;
+    var qCount = options.length;
+    var pick;
+    var index;
+    var newArray = [];
+    var holder = [];
+    
+    $("#reset").hide();
+    
+    $("#start").on("click", function () {
+            $("#start").hide();
+            displayQuestion();
+            runTimer();
+            for(var i = 0; i < options.length; i++) {
+        holder.push(options[i]);
+    }
+        })
+
+    function runTimer(){
+        if (!running) {
+        intervalId = setInterval(decrement, 1000); 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['decrement' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555496)